mm2src/coins/rpc_command/account_balance.rs

```diff
@@ -63,10 +63,10 @@ pub async fn account_balance(
     req: HDAccountBalanceRequest,
 ) -> MmResult<HDAccountBalanceResponseEnum, HDAccountBalanceRpcError> {
     match lp_coinfind_or_err(&ctx, &req.coin).await? {
-        MmCoinEnum::UtxoCoin(utxo) => Ok(HDAccountBalanceResponseEnum::Single(
```

Comment:
do we need the `::Single` variant anymore?

Reply:
Not really, just kept it for now in case we ever needed to switch back.